src/clib/lib/enkf/enkf_obs.cpp

```diff
@@ -1031,10 +1031,15 @@ void enkf_obs_load(Cwrap<enkf_obs_type> enkf_obs, const char *config_file,
     } catch (exc::out_of_range err) {
         if (enkf_obs->refcase) {
             throw exc::out_of_range(
-                "{}, the time map is set from the REFCASE keyword", err.what());
+                "{}, the time map is set from the REFCASE keyword.\n If the "
+                "date given to the observation is correct, You might have to "
+                "create a new refcase.",
```

Comment:
Suggest changing this to be more in line with the timemap statement? Find that to be more descriptive, and then we dont have to suggest a workaround that may or may not work.

Reply:
I am a bit unsure how you would like the message. Could you come with an example?

Reply:
The time map one was very descriptive, so for example:

```
Either the REFCASE has an incorrect/missing date, or the observation is given an incorrect date.
```

Reply:
Added :)

Reply:
Did you forget to push?

Reply:
pushed now :)